### ISSUE TITLE
Gestures for forward/backward history

### ISSUE THREAD

**Issue by orschiro** [wendorf/mousetures#1]
Hi @wendorf,

Your extension sounds very promising! 

Do you think it is possible to add gestures for going forward/backward in the search history of a tab?

Thankfully,

Robert

**Comment by wendorf** [comment]
Hi @orschiro,

Thanks for your interest in Mousetures. Forward/back in history is next in the priority list to implement. You can track the status of it in the [Navigate back in history on current tab](https://www.pivotaltracker.com/story/show/86858616) and [Navigate forward in history on current tab](https://www.pivotaltracker.com/story/show/86858618) stories in the backlog.

Cheers,
Dan

**Comment by orschiro** [comment]
@wendorf 

What great to read that you're considering this feature, thanks!

I am not using Pivotaltracker myself. Do you think you can announce it here when the feature is ready to be tested?

Thankfully,

Robert

**Comment by wendorf** [comment]
Yep, I'll comment and close this issue when it's completed and released.

**Comment by orschiro** [comment]
Thanks! :+1:
